app/assets/javascripts/discourse/app/controllers/preferences/profile.js

```diff
@@ -59,6 +58,16 @@ export default Controller.extend({
     return canChangeWebsite;
   },
 
+  @discourseComputed("model.can_upload_profile_header")
```

Comment:
Since the CP isn't modifying the input you should use the `alias` helper for these.

Reply:
said changes done